### DISCUSSION TITLE
Is there a way to calculate geographic distance between two points?

### DISCUSSION THREAD

**Discussion by tilltnet** [neocarto/geotoolbox#15] [General]
Is there currently a way to calculate the distance between two points in km/miles?

I think it would be cool to have a function in `geotoolbox` that works similarly to `st_distance()` in _simple features_ implementations like R's `sf` or SQL.

**Comment by BusraEcemSakar** [comment]
Yes, you can calculate the distance between two points in kilometres or miles in SQL.
You can use this query

SQL;

SELECT 
    ST_Distance(
        ST_SetSRID(ST_MakePoint(longitude1, latitude1), 4326), 
        ST_SetSRID(ST_MakePoint(longitude2, latitude2), 4326)
    ) AS distance_meters
FROM your_table;

**Comment by tilltnet** [comment]
Thank you so much for you response. I'm aware that this can be done in SQL. My questions was regarding how to do this in javascript with the `geotoolbox` library.
